WORKSPACE

```diff
@@ -50,7 +50,7 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 git_repository(
     name = "rules_sematic",
-    tag="v0.14.0",
+    tag="v0.22.1",
```

Comment:
do we want to add a test to update this for release?

Reply:
Yes, we would need an integration test that builds this pipeline and executes it against a server deployment. We should cover this in the upcoming integration test framework.